src/ec/packages/ec-component-expandable/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@ecl-twig/ec-component-expandable",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "0.0.1-alpha",
+  "description": "ECL Twig - EC Expandable",
+  "dependencies": {
+    "@ecl-twig/ec-component-button": "^0.0.1-alpha"
+  },
+  "devDependencies": {
+    "@ecl/ec-component-button": "^2.1.1",
```

Comment:
there is probably no need to have vanilla button as devDependency, as it is alreay the case in `ecl-twig/ec-component-button`